.github/workflows/ci.yml

```diff
@@ -660,6 +733,9 @@ jobs:
       - android_arm64
       - riscv32
       - riscv64
+
+      # Benchmark tools
+      - build_benchmark_tools
```

Comment:
What's next after this? `run_benchmarks_android` and `run_benchmarks_x86`? I'm wondering if those should go in a benchmarks.yml instead of ci.yml, and then how we'd want to orchestrate/trigger those jobs/tasks.

Reply:
Now that `workflow_call` is less limited, I think it would make sense to put these in `benchmarks.yml` with job names `android_arm64`, `x86_64`, `cuda`, etc. The UI will actually present these nicely

Reply:
I'll split the workflow in the follow-up PRs